submitit/helpers.py

```diff
@@ -239,6 +262,9 @@ def monitor_jobs(
     if not test_mode:
         assert sleep_time_s >= 30, "You can't refresh too often (>= 30s) to avoid overloading squeue"
 
+    job_arrays = ", ".join(sorted(set(str(job.job_id).split("_", 1)[0] for job in jobs)))
+    print(f"Monitoring job array {job_arrays} \n")
```

Comment:
Maybe add the number of jobs too ? "Monitoring {n} jobs from job arrays {job_arrays}"